example/App.js

```diff
@@ -22,15 +25,28 @@ const App = () => {
     hidePicker();
   };
 
+  const durationButton =
+    Platform.OS === "ios" ? (
+      <Button title="Show Duration Picker" onPress={showDurationPicker} />
+    ) : (
+      <View />
+    );
   return (
     <View style={style.root}>
       <Button title="Show Date Picker" onPress={showDatePicker} />
       <Button title="Show Time Picker" onPress={showTimePicker} />
+      {durationButton}
       <DateTimePickerModal
         isVisible={pickerMode !== null}
         mode={pickerMode}
         onConfirm={handleConfirm}
         onCancel={hidePicker}
+        headerTextIOS={`Pick a ${
+          pickerMode === "countdown"
+            ? "Duration"
+            : pickerMode &&
+              pickerMode.charAt(0).toUpperCase() + pickerMode.slice(1)
+        }`}
```

Comment:
I'd prefer to keep the example as simple as possible. I think we can skip showing the countdown mode here.